lib/routes/admin.js

```diff
@@ -34,6 +34,12 @@ router.route('/backups')
 router.route('/backups/:backup_guid/delete')
   .post(controller.handler('deleteBackup'))
   .all(middleware.methodNotAllowed(['POST']));
+router.route('/backups/:instance_id/summary')
```

Comment:
There is already an URL of the format `/backups/:backup_guid/delete` - generally one URL pattern identifies one resource type. So the in the above case, the portion of the URL `'/backups/:instance_id/` conflicts the regex of the above and it leads to ambiguity. So better to rename this to `/backups/summary/:instance_id`